validator/Makefile

```diff
@@ -68,13 +66,9 @@ validator:
 # We skip this for the development release
 DEVEL_RELEASE_IMAGE_VERSION ?= devel
 ifneq ($(strip $(VERSION)),$(DEVEL_RELEASE_IMAGE_VERSION))
-push-$(DEFAULT_PUSH_TARGET): push-short
+push-$(DEFAULT_PUSH_TARGET):
```

Comment:
Same as above comment. Can this be removed?

Reply:
Done